.github/workflows/krel-release-notes-validate.yaml

```diff
@@ -0,0 +1,140 @@
+# Copyright 2025 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Yaml Lint Release Notes
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - releases/**/release-notes/**.yaml
+      - releases/**/release-notes/**.yml
+  pull_request:
+    paths:
+      - releases/**/release-notes/**.yaml
+      - releases/**/release-notes/**.yml
+  # Allow manual triggering
+  workflow_dispatch: { }
+
```

Comment:
add

```
permissions: {}
```

this will drop all permissions for the workflow, as we dont need

Reply:
Added.